Sources/EpoxyCollectionView/CollectionView/CollectionView.swift

```diff
@@ -444,6 +444,8 @@ open class CollectionView: UICollectionView {
     cell.ephemeralViewCachedStateProvider = { [weak self] state in
       self?.ephemeralStateCache[item.dataID] = state
     }
+
+    cell.resetSelfSizingLayoutRecursionPreventionState()
```

Comment:
sanity check: do we think this is the only place we'd need to reset these values? I reset them here and in prepareForReuse... but I'd had to miss a spot and have a cell fail to size properly / keep using an old value.

Reply:
I don't think that should affect cell reuse, just `prepareForReuse` should do it as far as I understand. Might be worth tapping around some core screens just to be sure.